src/components/Checkbox/checkbox.scss

```diff
@@ -0,0 +1,135 @@
+$checkbox-transition: .2s cubic-bezier(0,0,.2,1) 0ms;
+
+.sb-checkbox {
+
+  &--inline {
+    position: relative;
+    display: inline-block;
+  }
+
+  &--outline {
+    border-radius: 0.5rem;
+    border: solid 0.1rem #eff1f5;
+    background-color: $white;
+    width: fit-content;
+    padding: 1.5rem 2rem;
+  }
+
+  &__inner {
+    position: relative;
+  }
+
+  &__native {
+    @include screenReaderOnly();
+
+    &:focus, &:hover {
+      &:not([disabled]) {
+        & + .sb-checkbox__input {
+          box-shadow: 0rem 0rem 0rem 0.2rem rgba(217, 244, 243, 1);
+        }
+      }
+    }
+    
+    &:checked {
+      & + .sb-checkbox__input {
+        border: solid 0.1rem $sb-green;
+        background-color: $sb-green;
+        
+        .sb-checkbox__truthy {
+          stroke-dashoffset: 0;
+          transition: stroke-dashoffset .2s cubic-bezier(.4,0,.6,1) 0ms;
+        }
+      }
+
+      &[disabled] {
+        & + .sb-checkbox__input .sb-checkbox__truthy {
+          stroke: #b1b5be;
+        }
+      }
+    }
+
+    &:not(:checked) {
+      & + .sb-checkbox__input {
+        border: solid 0.1rem $light;
+        background-color: $white;
+      }
+    }
+    
+    &:indeterminate {
+      & + .sb-checkbox__input {
+        border: solid 0.1rem $sb-green;
+        background-color: $sb-green;
+        
+        .sb-checkbox__indet {
+          transform: rotate(0) scale(1);
+          transition: transform $checkbox-transition;
+        }
+      }
+
+      &[disabled] {
+        & + .sb-checkbox__input .sb-checkbox__indet {
+          fill: #b1b5be;
+        }
+      }
+    }
+
+    &[disabled] {
+      & + .sb-checkbox__input {
+        border: solid 0.1rem $light;
+        background-color: #f5f5f5;
```

Comment:
Change the `background-color` to: `$white`;

Reply:
Disabled checkbox background isn't white!